src/scripts/debian.lua

```diff
@@ -109,7 +110,8 @@ function s.script(project)
       "love '"..loveFileDeb:gsub("'", "\\'").."'\n",
       true
   )
-  assert(fs.chmod(tempDir.."/usr/bin/"..project.package, "+x"))
+
+  assert(posix.chmod(tempDir.."/usr/bin/"..project.package, 'u+x'))
```

Comment:
Is it really `u+x` ? If the package is installed as root, then only root will be able to execute the game, right ?
I think that a better perm would be `rwxr-xr-x` (0755).